### ISSUE TITLE
Exponentiated tick labels not produced correctly

### ISSUE THREAD

**Issue by ndalchau** [jschwizer99/plot2svg#10]
Using Matlab 2015a.

The tick labels are not being produced correctly when the strings are of the format '10^3', as produced by default with logarithmically scaled axes. A simple test program reveals this:

```
loglog(10.^(0:3),10.^(0:3).*rand(1,4))
plot2svg('test.svg')
```

![test](https://cloud.githubusercontent.com/assets/7451485/10046125/d6f74c48-61fe-11e5-9749-2d5f4c29b890.png)

**Comment by leshaker** [comment]
same here...

**Comment by averter** [comment]
Same here. I am desperate. I really wanted to use svg on my thesis for the great quality. But these+inkscape bugs are sabotaging everything. Damn mathworks that have ruined everything since version R2014. I think I will have to go back to the old raster image figures. Truly  a pity.

**Comment by ndalchau** [comment]
Okay, I raised the issue a while ago, so I'm going to be a good citizen and propose a fix. More from me in due course.

**Comment by ndalchau** [comment]
FYI, I have proposed a fix in a pull request: #14 
If you need this functionality quickly, you can clone my Fork.

**Comment by kupiqu** [comment]
just wanted to let you know about the first release of fig2svg, which includes your fix among many other new features:

https://github.com/kupiqu/fig2svg
